husky_description/urdf/generate_dual_arm_husky_urdf.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
```

Comment:
It's not needed - but it's a nice to have. It can be generic, but I won't code it until someone needs it.